spec/mysql2/statement_spec.rb

```diff
@@ -320,7 +320,7 @@ def stmt_count
       result = @client.prepare("SELECT 1 UNION SELECT 2").execute(stream: true, cache_rows: false)
       expect do
         result.each {}
-        result.each {} # rubocop:disable Style/CombinableLoops
```

Comment:
This is still needed

Reply:
Indeed. Not sure why rubocop -a removed it. Maybe different version on my local.

Reply:
Ah, I see:

```
        ruby-version: 2.6
        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
```

It's using an old rubocop because rubocop dropped ruby 2.6 support a while ago.

Reply:
Hah, so we're cross-rubocop-versions too now 😉 even recently reducing the compat window, it's still quite wide